python/ray/tests/test_accelerated_dag.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+import logging
+import os
+import sys
+
+import numpy as np
+import pytest
+
+import ray
+import ray.cluster_utils
+
+logger = logging.getLogger(__name__)
+
+
+def test_put_local_get(ray_start_regular):
+    ref = ray._create_channel(1000)
+
+    num_writes = 1000
+    for i in range(num_writes):
+        val = i.to_bytes(8, "little")
+        ray._write_channel(val, ref, num_readers=1)
+        assert ray.get(ref) == val
+        ray._end_read_channel(ref)
+
+
+def test_put_different_meta(ray_start_regular):
```

Comment:
oh did you fix it, or they happen to have the same metadata size?

Reply:
I fixed it.